Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -508,6 +508,10 @@
 workflows[140.112] = ['',['RunCommissioning2022D','HLTDR3_2022','SKIMCOMMISSIONINGRUN3_reHLT_2022','HARVESTRUN3_2022']]
 workflows[140.113] = ['',['RunCosmics2022D','HLTDR3_2022','SKIMCOSMICSRUN3_reHLT_2022','HARVESTRUN3_COS_2022']]
 
+### run3 (2022) reMINIAOD+NANO ###
+workflows[140.201] = ['',['RunJetMET2022D_reMINI', 'REMINIAOD_data2022']]
```

Comment:
Why do we need only MINI workflow, and MINI+Nano+DQM?

Reply:
MINI + Validation +  DQM:@miniAOD fails with an error of missing HGCal collections. I'm really puzzled by the error and do not have time to debug the DQM side at this stage. The MINI and NANO content are anyway as expected, therefore I think the issue with DQM can be dealt with later

Reply:
Yeah, I have that question too on DQM. Since you start from RECO, I am not sure DQM will work as you may miss some transient product which are produced during reconstruction. May I propose to drop the broken workflow for now, for example, you comment it. This will avoid broken in the long matrix test in IB, as workflow in the standard will run. Thx.

Reply:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12984e/34739/runTheMatrixINPUT-results/140.201_RunJetMET2022D_reMINI/

Reply:
Ah, OK. I read it quick and confuse on DQM module.
The DQM of Mini should fail as they need transient output from RECO, but DQM of Nano does not need. 

What I don't see the point is why we need both workflows as both produce the same MINI.

Reply:
There was some concern (I just unjustified) that running MINI or MINI+NANO in the same job could lead to different results / issues. I agree with you that the MINI only workflow is not strictly needed. I will revise that with a follow up PR (we, xpog, want to revise some validation workflows anyway).